### DISCUSSION TITLE
default angle of the model

### DISCUSSION THREAD

**Discussion by tom-deadline** [google/model-viewer#2135] [Q&A] [answered]
Hey, is there a way to set the default angle of the model? So when it loads it is loaded with a specific angle.

**Comment by milesgreen** [comment, marked as answer]
You can use the `orientation` attribute:
https://modelviewer.dev/docs/#entrydocs-scenegraph-attributes-orientation

You may also need to call `updateFraming()` method, depending on when you change `orientation`. 

Demo:
https://modelviewer.dev/examples/scenegraph/#transforms

**Reply by tom-deadline** [reply]
@milesgreen Thank you for your help, everything worked.
